### PR TITLE
Morph base app UI to resemble final product more closely

### PR DESCRIPTION
**Pull request by Aulud**
## Morph base app UI to resemble final product more closely
Summary: Implements changes to the basic UI of AB3 to set the canvas for subsequent design of GUI elements for new commands. Addresses #60. This supersedes PR #54, which was merged erroneously earlier.

### Changelog
- Resize app dimensions to 384px (H) by 700px (H) for mobile form factor
- Suppress resizing of app window; force default window size on startup regardless of GUI preferences
- Redesign and style base GUI of app component-wise to a light theme
   - Remove redundant header menu and all references to menu UI elements
   - Reduce font sizes; force result display label to wrap text
- Separate CSS stylesheets by category into one of `LightTheme.css`, `Components.css` or `Extensions.css`

#### Before-after comparison
Splitwiser-comparison

Last updated 15 Oct 2019, 11:00PM

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -1,59 +1,35 @@
...
-<?import javafx.scene.control.MenuBar?>
-<?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.SplitPane?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="SplitWiser" minWidth="384" maxWidth="384" minHeight="700" maxHeight="700" onCloseRequest="#handleExit">
```
Comment (daekoon) [inline src/main/resources/view/MainWindow.fxml]: Quick question, is there really a reason on why we are changing the window size to a mobile form factor? Its currently not a mobile app, and I feel like the window can be slightly larger to display more info. I can see stuff like view Activity needing more space, and we already have pretty long error messages that require users to scroll up/down to see the full error message.

Comment (daekoon) [reply src/main/resources/view/MainWindow.fxml]: Oh well, I guess its a small change that can be done quickly in the future. Guess we are sticking with this change

**Review by daekoon — DISMISSED** [review]
LGTM

File: src/main/resources/view/MainWindow.fxml (hunk 1 above)
Comment (Aulud) [reply src/main/resources/view/MainWindow.fxml]: Have increased the app width to give us more canvas space to mess around with stuff :)

**Review by daekoon — APPROVED** [review]
LGTM

**Pull request merged**